structs-benchmark/build.gradle

```diff
@@ -24,9 +24,16 @@ applicationDefaultJvmArgs = ['-XX:+UnlockCommercialFeatures', '-XX:+UnlockDiagno
 compileKotlin {
     kotlinOptions.jvmTarget = "1.8"
     kotlinOptions.freeCompilerArgs = ["-Xno-call-assertions", "-Xno-param-assertions"]
+    // A temporary fix for KT-41105 to circumvent a compiler bug where using a delegated property with inline operators
+    // causes the compiler to crash
+    kotlinOptions.useIR = true
```

Comment:
That was because KT-41105 is a bug where the compiler crashes saying that it can't find the getValue and setValue operators whenever they are inline. IR seems to be causing problems however, so I'll make the operators not inline again and hopefully it'll work with the normal backend